docs/source/changelog.rst

```diff
@@ -13,6 +13,17 @@ Changelog
 .. Bug Fixes
 .. +++++++++
 
+0.13.1 / 2020-02-05
+-------------------
+
+New Features
+++++++++++++
+- (:pr:`209`) Added the option to Hill-order molecular formulas.
+
+Bug Fixes
++++++++++
+- (:pr:`208`) Fixes a Molecule hashing issue due to order of operations changes in ``Molecule.from_data``.
```

Comment:
The order of operations changed in ``Molecule.from_data`` and occasionally resulted in different hashes for Molecules undergoing `orient` operations. This issue was introduced in 0.13.0 and is unlikely to have any serious negative effects as this did not effect hash integrity.